java/arcs/core/storage/Handle.kt

```diff
@@ -69,7 +69,7 @@ open class Handle<Data : CrdtData, Op : CrdtOperationAtTime, T>(
     val storageProxy: StorageProxy<Data, Op, T>,
 
     /** [callback] contains optional Handle-owner provided callbacks to add behavior. */
-private val callback: Callbacks<Data, Op, T>? = null,
+    var callback: Callbacks<Data, Op, T>? = null,
```

Comment:
Why this change, is this ok long-term or an interim change, should we update docs?

Reply:
It originally was public mutable when I wrote my PR, then after I rebased, someone changed it to private immutable. I can change it back, but it means making a field in the Adapter class mutable, because StorageHandles have to be created before their SdkHandle wrapper, but the Callbacks adapter needs to reference SdkHandle in its ctor. There's no topological sort that works. :)

I can add a SetHandle() to the SenderCallbacksAdapter and modify it afterwards.

Reply:
I think we can switch this in a followup.